src/formatSeconds/index.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Given an object conatining totalSeconds, return the minutes and seconds in the
+ * format MM:SS, it will pad 0s to ensure we always have 4 digits, e.g 01:01
+ * @param {*} Object totalSeconds
+ * @returns String MM:SS
```

Comment:
@returns can also have a type specified like @param, so it would be `@returns {string} MM:SS`